scalafmt-tests/src/test/resources/newlines/source_keep.stat

```diff
@@ -4192,27 +4200,28 @@ type T = A1 & A2 | A3 & A4 | A5 & A6 /* c1 */ | /* c2 */ A7 & A8 | A9 & A10[T10]
   A51 & A52 | A53 & A54 | A55 & A56 /* c1 */ | /* c2 */ A57 & A58 | A59 & A60[T60] |
   A61 & A62 | A63 & A64 | A65 & A66 /* c1 */ | /* c2 */ A67 & A68 | A69 & A70[T70]
 >>>
-type T = A1 & A2 | A3 & A4 | A5 &
-  A6 /* c1 */ | /* c2 */ A7 & A8 | A9 &
-  A10[T10] |
-  A11 & A12 | A13 & A14 | A15 &
-  A16 /* c1 */ | /* c2 */ A17 & A18 |
-  A19 & A20[T20] |
-  A21 & A22 | A23 & A24 | A25 &
-  A26 /* c1 */ | /* c2 */ A27 & A28 |
-  A29 & A30[T30] |
-  A31 & A32 | A33 & A34 | A35 &
-  A36 /* c1 */ | /* c2 */ A37 & A38 |
-  A39 & A40[T40] |
-  A41 & A42 | A43 & A44 | A45 &
-  A46 /* c1 */ | /* c2 */ A47 & A48 |
-  A49 & A50[T50] |
-  A51 & A52 | A53 & A54 | A55 &
-  A56 /* c1 */ | /* c2 */ A57 & A58 |
-  A59 & A60[T60] |
-  A61 & A62 | A63 & A64 | A65 &
-  A66 /* c1 */ | /* c2 */ A67 & A68 |
-  A69 & A70[T70]
+type T =
+  A1 & A2 | A3 & A4 | A5 & A6 /* c1 */ | /* c2 */ A7 & A8 | A9 & A10[
```

Comment:
while this might look bad, in reality it's because in the original code snippet, there were no breaks between infix types. for any existing code which had previously been formatted using source=keep and Type.Or trees, the modified code wouldn't change formatting because it would still preserve breaks.